code/FulpstationCode/fulpops.dm

```diff
@@ -0,0 +1,32 @@
+/datum/outfit/syndicate/plasmaman
+	name = "Syndicate Operative Plasmaman - Basic"
+
+	head = /obj/item/clothing/head/helmet/space/plasmaman/robotics/syndicate
```

Comment:
why robotics and security? is there no plasmaman/syndicate one?

Reply:
Yeah syndicate don't have their own custom plasmaman outfit so I just copied over the normal robotics one which was black and red.

Reply:
If it's just a question of aesthetics I think that's fine.